### DISCUSSION TITLE
about github

### DISCUSSION THREAD

**Discussion by spartaverse** [dendrovis/github#8] [General]
what is a github

**Comment by dendrovis** [comment]
hi @spartaverse you can refer to https://github.com/features for the details. In short, github is a place to manage code in remotely and open for collaborations with other developer

**Reply by lucas-0liveira** [reply]
yes, you can

**Reply by Flocons12** [reply]
Je veux cloner le compte WhatsApp de ma femme 224620120914 aidez moi
